### PR TITLE
fix: include assets in release

### PR DESCRIPTION
**Pull request by skyerus**
## This PR

Automatic upload of assets after a release.

### Related Issues

Fixes https://github.com/open-feature/open-feature-operator/issues/108

### CODE REVIEW

**Comment by beeme1mr** [comment]
I manually uploaded the missing assets.

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -16,3 +16,35 @@ jobs:
...
+
+  release-assets:
+    needs: release-please
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            submodules: recursive
```
Comment (beeme1mr) [inline .github/workflows/release-please.yml:31]: I'm not sure if we need the submodule anymore. Also, you may want to checkout the tagged version like we did in the Go SDK.

https://github.com/open-feature/go-sdk/blob/main/.github/workflows/release-please.yml#L35

Comment (skyerus) [reply .github/workflows/release-please.yml]: done

**Review by beeme1mr — APPROVED** [review]
Looks good, thanks!

**Review by AlexsJones — APPROVED** [review]

**Pull request merged**